build-rkt

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+export SRC_DIR=`pwd`
+
+sudo rkt run \
+    --volume src-dir,kind=host,source=${SRC_DIR} \
+    --volume build-dir,kind=host,source=${SRC_DIR} \
+    --interactive \
```

Comment:
The `--interactive` argument is not needed here, as the build process will not require human interaction. I hope I'm not wrong with the latter ;-)